docs/dev/remote-coding-environments.md

```diff
@@ -0,0 +1,29 @@
+# Remote Coding Environments For Gaffer
+
+Gaffer is now configured for remote coding environemnts such as GitHub
+Codespaces and Gitpod. This addition allows for an easier and faster way to
+contribute to Gaffer with no manual setup or need to download dependices to a
+local machine.
+
+## GitHub Codespaces
+To use GitHub Codespaces simply open the gaffer repsoitory and click the "Code"
+button drop down. You'll see an option labeled "Open with codespaces" Click on
+it.
```

Comment:
```suggestion
To use GitHub Codespaces simply open the Gaffer repository, click the "Code" button
drop down, and then the option labeled "Open with Codespaces".
```
However, this is not what I see. I see a tab called codespaces and a green "Create codespace on develop" button.